### PR TITLE
ENH: Adds base code to make the package pip-installable

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

**Comment by ElDeveloper** [comment]
With this we should be able to then add travis, nose, etc.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+classes = """
+    Development Status :: 1 - Planning
+    License :: OSI Approved :: BSD License
+    Topic :: Software Development :: Libraries
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
```
Comment (mortonjt) [inline setup.py]: We're dropping python2?

Comment (ElDeveloper) [inline setup.py]: ¯_(ツ)_/¯ ... not sure at all, good thing to discuss as a group.

Comment (mortonjt) [inline setup.py]: I thought we already discussed this as a group?  @antgonza @josenavas , do you guys remember?

Comment (antgonza) [inline setup.py]: Well, ideally we will support both but perhaps for simplicity we should focus on 3 ...

Comment (ElDeveloper) [inline setup.py]: @mortonjt, I don't recall that conversation, my bad. We can do 3-only
then. Anyone has objections?

Comment (mortonjt) [inline setup.py]: I don't have any objections.

Comment (josenavas) [inline setup.py]: 3-only sounds good to me

**Comment by josenavas** [comment]
@ElDeveloper do you think it is worth to add .travis.yml here so it actually already checks that the setup.py works correctly?

File: setup.py (hunk 1 above)
Comment (wasade) [inline setup.py]: :+1: on 3 only

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Operating System :: Unix
```
Comment (wasade) [inline setup.py:32]: no 3.5?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      author="mdsa development team",
+      author_email="antgonza@gmail.com",
+      maintainer="mdsa development team",
+      maintainer_email="antgonza@gmail.com",
+      url='http://github.com/biocore/mds-approximations/',
+      test_suite='nose.collector',
+      packages=find_packages(),
+      setup_requires=['numpy >= 1.9.2'],
+      install_requires=[
+          'decorator >= 3.4.2',
```
Comment (wasade) [inline setup.py]: why all these bits?

**Comment by ElDeveloper** [comment]
Yeah, I will do that, seems reasonable!

File: setup.py (hunk 3 above)
Comment (ElDeveloper) [inline setup.py]: Vestigial of the original setup.py - removing now!

**Comment by ElDeveloper** [comment]
Should be ready for another review.

**Comment by mortonjt** [comment]
Looks good to me! Ready to merge?

**Comment by ElDeveloper** [comment]
Thanks guys!

**Pull request merged**